modules/mono/editor/GodotTools/GodotTools/Ides/Rider/RiderPathManager.cs

```diff
@@ -46,7 +47,7 @@ public static void Initialize()
                 }
 
                 var riderPath = (string)editorSettings.GetSetting(EditorPathSettingName);
-                if (IsRiderAndExists(riderPath))
+                if (Exists(riderPath))
```

Comment:
```suggestion
                if (File.Exists(riderPath))
```
Same for the other use and remove the method.